packages/manager/modules/pci/src/projects/project/kubernetes/details/node-pool/add/translations/Messages_fr_FR.json

```
@@ -9,7 +9,7 @@
   "kube_add_node_pool_error_InsufficientPortsQuota": "Le quota de ports de votre projet est insuffisant pour réaliser l'opération. Merci d'augmenter votre quota ou demander moins de ressources.",
   "kube_add_node_pool_error_InsufficientInstancesQuota": "Le quota d'instances de votre projet est insuffisant pour réaliser l'opération. Merci d'augmenter votre quota ou demander moins de ressources.",
   "kube_add_node_pool_error_quota_link": "Augmenter mon quota.",
-  "kube_add_node_pool_name_input_pattern_validation_error": "Le nom doit commencer par un caractère alphabétique, se terminer par un caractère alphanumérique et les seuls caractères spéciaux autorisés sont les tirets et les underscores.",
+  "kube_add_node_pool_name_input_pattern_validation_error": "Le nom doit plus d'un caractère, commencer par un caractère alphabétique, se terminer par un caractère alphanumérique et les seuls caractères spéciaux autorisés sont les tirets et les underscores.",
```

i guess an issue regarding the phrase; i guess this is better:
```suggestion
  "kube_add_node_pool_name_input_pattern_validation_error": "Le nom doit être composé au moins d'un caractère, commencer par un caractère alphabétique, se terminer par un caractère alphanumérique et les seuls caractères spéciaux autorisés sont les tirets et les underscores.",
```